docs/windows.md

```diff
@@ -426,6 +357,128 @@ antrea-agent-windows-6hvkw                             1/1     Running     0
 kube-proxy-windows-2d45w                               1/1     Running     0          102s
 ```
 
+### Add Windows kube-proxy DaemonSet (only for Kubernetes versions prior to 1.26)
+
+Windows kube-proxy will no longer be supported by Antrea starting from Kubernetes 1.26. This decision is based on the deprecation of the userspace mode,
+which Antrea heavily relies on. You should skip this step on the latest kubernetes cluster and enable proxy-all mode for windows agent.
+
+Add Windows-compatible versions of kube-proxy by applying file `kube-proxy.yaml`.
+
+Download `kube-proxy.yaml` from kubernetes official repository and set
+kube-proxy version.
+
+```bash
+# Example:
+curl -L "https://github.com/kubernetes-sigs/sig-windows-tools/releases/download/v0.1.5/kube-proxy.yml" | sed 's/VERSION/v1.23.5/g' > kube-proxy.yml
```

Comment:
maybe use the latest supported version instead of `v1.23.5`?

Reply:
I found the yaml file has been updated and the image is VERSION-nanoserver now, which doesn't support our current kube-proxy script, the latest available image version is v1.20.0.